swssconfig/sample/00-copp.config.json

```diff
@@ -50,8 +50,8 @@
             "queue": "1",
             "meter_type":"packets",
             "mode":"sr_tcm",
-            "cir":"600",
-            "cbs":"600",
+            "cir":"6000",
```

Comment:
I think this can make the cpu/kernel busy if there is any packets like ping destined to switch. I would assume it was set to 600pps to control such packets. How about setting it to 1000pps instead of increasing it 10x.

Reply:
Code downloads are the pain point. With 1000pps we will see speeds ~ 1.4MB/s and it takes 7 minutes to download a full sonic image compared to 8.4MB/s with 6000pps. I think 6000pps is not that bad. Most of the switches are having an intel CPU and 8.4MB/s b/w for in-band access looks like a number on the conservative side.

Reply:
@rajendra-dendukuri what about keep the current value as 600 and leave the user to change the setting?

Reply:
@stcheng  Are you suggesting that end user changes the setting in source code and re-compile's an image?